src/app/(shop)/[page]/page.tsx

```diff
@@ -0,0 +1,89 @@
+import Page from '@/features/Page/Page';
+import { getAllPageSummaryItems } from '@/features/Page/data';
+import { PageGetPage } from '@/features/Page/queries';
+import { getPage, getPageParams } from '@/features/Page/transforms';
+import { getAnonymousTakeshapeClient } from '@/lib/apollo/rsc';
+import { ServerProps } from '@/types/next';
+import { PageGetPageResponse, PageGetPageVariables } from '@/types/takeshape';
+import { ApolloError } from '@apollo/client';
+import * as Sentry from '@sentry/nextjs';
+import { Metadata } from 'next';
+import { notFound } from 'next/navigation';
+
+export const revalidate = 300;
```

Comment:
`revalidate` must be statically defined, so unfortunately can't use a variable for these... Most pages are 5 mins, homepage is 1 min.

This also effects how long data is cached during static generation — so for instance navigation data would get fetched a couple times during a 10 minutes build.